### DISCUSSION TITLE
How to donate to or sponsor this project?

### DISCUSSION THREAD

**Discussion by stairjoke** [MonitorControl/MonitorControl#771] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

This app is amazing and it's going to die eventually.

### Describe the solution you'd like

I'd like to pay for it so it stays alive.

### Describe alternatives you've considered

Teaching myself to code and contributing to the project.

### Anything else?

Patreon?

**Comment by waydabber** [comment, marked as answer]
Hey @stairjoke

Thanks for the question and care! Currently the only way to donate is through OpenCollective:

https://opencollective.com/monitorcontrol/donate

You can do a one time donation or become a backer with a monthly commitment.

You can even become a Bronze/Silver/Gold Sponsor (so far we have managed to gather zero Sponsors lol) and depending on the level of Sponsorship, there are some additional honors given. :)

https://opencollective.com/monitorcontrol

We hope MonitorControl won't die but it is true that the enthusiasm to develop it might come in (rather unpredictable) waves - the pace of improvement (and support time) displayed in the past months is hardly sustainable on the long run (but the app is already packed with features and there was kind of a backlog, so I don't think this is much of an issue). It is true that compared to the effort and development time poured into it there is minimal monetary return - which is completely fine, I think nobody expects to get rich out of a free and opensource project. Maybe some lucky guy can manage that but for us that would be an unexpected turn of events. We do this for fun + the satisfaction that others find the tool we create useful. :))

**Comment by MacDada** [comment]
Please add crypto address for donations :-)

**Reply by the0neyouseek** [reply]
You can donate using crypto on our OpenCollective page 😃 <https://opencollective.com/monitorcontrol/donate/crypto>
